src/indexer/Cargo.toml

```diff
@@ -0,0 +1,15 @@
+[workspace]
+exclude = ["processor"]
```

Comment:
Why?

Reply:
The processor is already in a workspace inside the aptos directory. I gives an error if we add it to the workspace. Can't wait to try the SDK though